Terminal.Gui/MainLoop.cs

```diff
@@ -105,6 +105,11 @@ public void Dispose ()
     ///     </para>
     /// </remarks>
     /// <param name="idleHandler">Token that can be used to remove the idle handler with <see cref="RemoveIdle(Func{bool})"/> .</param>
+    // QUESTION: Why are we re-inventing the event wheel here?
```

Comment:
This is original gui.cs code and we've never cared enough to re-engineer it. Probably time. ;-)

Reply:
Cool. I thought as much. This will likely get nuked during the event refactor, since it's just events with more steps. :)